vulture_os/toolkit/api_parser/sentinel_one_identity_graph/sentinel_one_identity_graph.py

```diff
@@ -0,0 +1,263 @@
+#!/home/vlt-os/env/bin/python
+"""This file is part of Vulture OS.
+
+Vulture OS is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Vulture OS is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Vulture OS.  If not, see http://www.gnu.org/licenses/.
+"""
+__author__ = "Theo Bertin"
+__credits__ = []
+__license__ = "GPLv3"
+__version__ = "4.0.0"
+__maintainer__ = "Vulture OS"
+__email__ = "contact@vultureproject.org"
+__doc__ = 'Proofpoint TAP SIEM API'
+__parser__ = 'PROOFPOINT'
+
+import logging
+from datetime import datetime, timedelta
+from json import JSONDecodeError, dumps
+from requests.exceptions import ConnectionError, ConnectTimeout, HTTPError, ReadTimeout
+from requests import Session
+
+from django.conf import settings
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+from toolkit.api_parser.api_parser import ApiParser
+
+
+logging.config.dictConfig(settings.LOG_SETTINGS)
+logger = logging.getLogger('api_parser')
+
+
+class SentinelOneIdentityGraphError(Exception):
+    pass
+
+
+class SentinelOneIdentityGraphParser(ApiParser):
+    def __init__(self, data):
+        self.sentinel_one_identity_graph_token = data["sentinel_one_identity_graph_token"]
+
+        self.sentinel_one_identity_graph_console_url = data["sentinel_one_identity_graph_console_url"]
```

Comment:
you should sanitize the value, ensure scheme is there (and only once), and strip-add single '/' at the end